### DISCUSSION TITLE
没有一条评论的文章 显示【未找到相关的Issues进行评论 请联系@xxx 初始化创建】

### DISCUSSION THREAD

**Discussion by suntingtingcslg** [gitalk/gitalk#493] [Q&A]
还未有一条评论的文章，在请求https://api.github.com/repos/username/hexo-comment/issues?labels=Gitalk,14e80fe5b434086704210194c668158d&t=1638270795351接口时候 ，返回一个空数组 ，然后就提示【未找到相关的Issues进行评论 请联系@xxx 初始化创建】。没有写第一条评论的入口，就无法写入评论。截图如下
<img width="1382" alt="截屏2021-11-30 下午7 09 25" src="https://user-images.githubusercontent.com/32606101/144038111-0ce992de-2fcb-4b4e-8074-d2e7ac788c1b.png">
<img width="1382" alt="截屏2021-11-30 下午7 09 32" src="https://user-images.githubusercontent.com/32606101/144038110-e0b3785d-2eb4-43c0-bbbf-9eadb44b3a4a.png">
<img width="424" alt="截屏2021-11-30 下午7 11 59" src="https://user-images.githubusercontent.com/32606101/144038109-a320df0a-4147-473f-acfd-3fd972c64d8d.png">
没有一条评论的文章，也显示写评论的页面呢？
ps：gitalk版本是1.7.2

**Comment by helloint** [comment]
需要你用shijiehehai这个GitHub账号登录，并访问这篇文章至少一次，进行对评论的初始化，或者说，这样API才有权限创建这个GitHub Issue
